management/server/http/api/openapi.yml

```diff
@@ -913,6 +913,10 @@ components:
           description: The ID of the initiator of the event. E.g., an ID of a user that triggered the event.
           type: string
           example: google-oauth2|123456789012345678901
+        initiator_email:
+          description: The e-mail address of the initiator of the event. E.g., an e-mail of a user that triggered the event.
+          type: string
```

Comment:
Good idea, I will extend with it.

Reply:
Unfortunately I have to say I can not use your suggestion because it depend on multiple library update and it can cause unpredictable issues. It is too much risk at this point.

Reply:
Oh, I see. No need to do this now then. Thank you for verifying!